spec/queries/get_change_offer_options_spec.rb

```diff
@@ -0,0 +1,131 @@
+require 'rails_helper'
+
+RSpec.describe GetChangeOfferOptions do
+  include CourseOptionHelpers
+
+  let(:ratifying_provider) { create(:provider) }
+  let(:self_ratified_course) { create(:course, :open_on_apply, provider: ratifying_provider) }
+  let(:externally_ratified_course) { create(:course, :open_on_apply, accredited_provider: ratifying_provider) }
+  let(:provider_user) { create(:provider_user) }
+
+  def service(provider_user, course)
+    described_class.new(user: provider_user, current_course: course)
+  end
+
+  def set_provider(provider, make_decisions: true)
+    provider_user
+      .provider_permissions
+      .find_or_create_by(provider: provider)
+      .update(make_decisions: make_decisions)
+  end
+
+  def set_provider_permissions(training_provider, make_d1, ratifying_provider, make_d2)
```

Comment:
Nits: maybe `set_make_decisions_permissions` for clarity?

Reply:
Changed!